Patches/python.diff

```diff
@@ -287,84 +240,39 @@ diff -r 8667c26e2bec Python/ceval.c
  #ifdef DXPAIRS
      int lastopcode = 0;
  #endif
-@@ -1170,7 +1195,7 @@
- 
-        f->f_lasti now refers to the index of the last instruction
-        executed.  You might think this was obvious from the name, but
--       this wasn't always true before 2.3!  PyFrame_New now sets
-+       this wasn't always true before 2.3!  PyFrame_New now setsYIELD_
```

Comment:
Another case of removing changes - that YIELD_ slipped in at some point and shouldn't have been there